tests/unit/Application/Metadata/JsonGeneratorTest.php

```diff
@@ -487,7 +487,8 @@ public function test_it_builds_an_entity_change_request()
 
         $contact = m::mock(Contact::class);
         $contact->shouldReceive('getEmailAddress')->andReturn('j.doe@example.com');
-        $data = $generator->generateEntityChangeRequest($entity, $diff, $contact);
+        $contact->shouldReceive('getDisplayName')->andReturn('A.F.Th. van der Heijden');
```

Comment:
Very specific name in the test :-)

Reply:
Always nice to thow in a couple of freak references to literature here and there.